cherrypy/test/test_request_obj.py

```diff
@@ -761,7 +761,7 @@ def test_basic_HTTPMethods(self):
         # Provide a C-T or webtest will provide one (and a C-L) for us.
         h = [("Content-Type", "text/plain")]
         self.getPage("/method/reachable", headers=h, method="PUT")
-        self.assertStatus(411)
```

Comment:
This change isn't appropriate. The test is explicitly testing the server to show that it returns a 411 Content Length Required when no content length is supplied.